File: Client/src/app/components/models/stock-list.model.ts
import { IActualStockModel } from "./stock-model/actual-stock.model";
import { CalculationType } from "./stock-model/stock-base.model";

export interface IStockListModel {
  Id: string;
  Title: string;
  Stocks: IActualStockModel[];
  IsNotificated: boolean;
  CalculationType: CalculationType;
  Ratio: number;
}

File: Backend/Models/Client/StockListModel.cs
﻿using Backend.Models.Client.StockModel;

namespace Backend.Models.Client
{
    public class StockListModel : Entity
    {
        public string Title { get; set; } = string.Empty;

        public List<ActualStockModel> Stocks { get; set; } = new List<ActualStockModel>();

        public bool IsNotificated { get; set; } = true;

        public CalculationType CalculationType { get; set; } = CalculationType.Hormonic;

        public int Ratio { get; set; } = 2;
    }
}
